### PR TITLE
feat(docker): Add build var for optionally installing abiword

### PR DESCRIPTION
**Pull request by neurolabs**
This MR introduces a docker build variable `INSTALL_ABIWORD`. When set
to any value other than `0`, ABIWORD is installed in the resulting
docker container, enabling the possibility to configure ABIWORD in
settings.json.docker or via ENV VAR `ABIWORD` for exporting to
DOC/PDF/ODT.

Documentation is included inline and in the docker markdown file.

### CODE REVIEW

File: doc/docker.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ The variable value has to be a space separated, double quoted list of plugin nam
 
 Some plugins will need personalized settings. Just refer to the previous section, and include them in your custom `settings.json.docker`.
 
+### Rebuilding including export functionality for DOC/PDF/ODT
+If you want to be able to export your pads to DOC/PDF/ODT files, you can set the INSTALL_ABIWORD build variable to any value other than "0".
+
+Also, you will need to configure the path to the abiword executable (`/usr/bin/abiword`) via editing `<BASEDIR>/settings.json.docker` or via setting the environment variable `ABIWORD` to `/usr/bin/abiword`.
```
Comment (rhansen) [inline doc/docker.md:35]: I know that the rest of the file isn't line-wrapped, but I would prefer it to be:

```suggestion
### Rebuilding including export functionality for DOC/PDF/ODT

If you want to be able to export your pads to DOC/PDF/ODT files, you can set the
INSTALL_ABIWORD build variable to any value other than "0".

Also, you will need to configure the path to the abiword executable
(`/usr/bin/abiword`) via editing `<BASEDIR>/settings.json.docker` or via setting
the environment variable `ABIWORD` to `/usr/bin/abiword`.
```

**Review by rhansen — COMMENTED** [review]
Thanks for this!

File: Dockerfile [hunk 2]
```diff
@@ -15,6 +15,14 @@ LABEL maintainer="Etherpad team, https://github.com/ether/etherpad-lite"
 #   ETHERPAD_PLUGINS="ep_codepad ep_author_neat"
 ARG ETHERPAD_PLUGINS=
 
+# Control whether abiword will be installed, enabling exports to DOC/PDF/ODT formats.
```
[the remark this answers is not in the document]
Comment (neurolabs) [reply Dockerfile:18]: You tell me ;) I was under the impression that the project prefers Abiword. But probably it's good to keep the option of choosing between Abiword and Libreoffice. I will do that using two build variables, one additional one for libreoffice.

**Review by neurolabs — COMMENTED** [review]
Thanks for the review!

**Comment by neurolabs** [comment]
OK, ready for another review.

I added support and documentation for libreoffice analogous to abiword.

**Comment by neurolabs** [comment]
@rhansen want to take another look?

**Review by rhansen — APPROVED** [review]
Thank you!

**Pull request merged**